src/RoslynInsertionTool/RoslynInsertionTool/LegacyInsertionArtifacts.cs

```diff
@@ -32,14 +32,14 @@ public static bool TryCreateFromLocalBuild(string buildDirectory, out InsertionA
         public override string GetPackagesDirectory()
         {
             // For example: "\\cpvsbuild\drops\Roslyn\Roslyn-Main-Signed-Release\20160315.3\DevDivPackages"
```

Comment:
This comment should not apply anymore right, since we can't read from here

Reply:
I'm not sure when this comment would have applied. Even when working with legacy artifacts today they are either downloaded as part of the pipeline trigger or we download them from the build. We don't try to access them from a share.